src/fides/data/privacy_notices/privacy_notice_templates2.yml

```diff
@@ -0,0 +1,470 @@
+
+- name: Sensitive Personal Data Sharing
+  notice_key: sensitive_personal_data_sharing_gpp_utah
+  id: pri_36d70a75-1a3a-4682-8da4-4698df1a960e_spds_gpp_utah
+  internal_description: Sensitive personal data sharing (GPP, Utah)
+  consent_mechanism: opt_out
+  # data_uses: data uses must be applied manually
+  enforcement_level: not_applicable
+  disabled: True
+  has_gpc_flag: False
+  framework: gpp_us_state
+  translations:
+    - language: en_us
+      title: Sensitive Personal Data Sharing
+      description: We may transfer or share Sensitive Personal Data about you with third parties. You can learn more about how this information is used in our privacy notice.
+  gpp_field_mapping:
+    - region: us_ut
+      notice:
+        - SensitiveDataProcessingOptOutNotice
+      mechanism:
+        - field: SensitiveDataProcessing
+          not_available: "00000000"
+          opt_out: "11111111"
+          not_opt_out: "22222222"
+
+- name: Sensitive Personal Data Sharing
+  notice_key: sensitive_personal_data_sharing_gpp_us_state_opt_in
+  id: pri_f46d06be-6b40-472c-86da-2c910e7aa800_spds_gpp_state_opt_in
+  internal_description: Sensitive personal data sharing (GPP, state-by-state, opt-in)
+  consent_mechanism: opt_in
+  # data_uses: data uses must be applied manually
+  enforcement_level: not_applicable
+  disabled: True
+  has_gpc_flag: False
+  framework: gpp_us_state
+  translations:
+    - language: en_us
+      title: Sensitive Personal Data Sharing
+      description: We may transfer or share Sensitive Personal Data about you with third parties. You can learn more about how this information is used in our privacy notice.
+  gpp_field_mapping:
+    - region: us_co
+      notice:
+      mechanism:
+        - field: SensitiveDataProcessing
+          not_available: "0000000"
+          opt_out: "1111111"
+          not_opt_out: "2222222"
+    - region: us_ct
+      notice:
+      mechanism:
+        - field: SensitiveDataProcessing
+          not_available: "00000000"
+          opt_out: "11111111"
+          not_opt_out: "22222222"
+    - region: us_va
+      notice:
+      mechanism:
+        - field: SensitiveDataProcessing
+          not_available: "00000000"
+          opt_out: "11111111"
+          not_opt_out: "22222222"
+
+- name: Known Child Sensitive Data Consents
+  notice_key: known_child_sensitive_data_consents_gpp_us_state
+  id: pri_0c8f3012-c37a-4478-9d1a-5a2f807c89ea_kcsdc_gpp_state
+  internal_description: Known child sensitive data consents (GPP, state-by-state)
+  consent_mechanism: opt_in
+  # data_uses: data uses must be applied manually
+  enforcement_level: not_applicable
+  disabled: True
+  has_gpc_flag: False
+  framework: gpp_us_state
+  translations:
+    - language: en_us
+      title: Known Child Sensitive Data Consents
+      description: We may process the personal information of children under 16. You can learn more in our privacy policy.
+  gpp_field_mapping:
+    - region: us_ca
```

Comment:
a _little bit_ odd that regions are referenced here in this mapping even though regions are no longer directly on the notice, but i think that's alright!

Reply:
I agree these still feel like notice-specific details